### PR TITLE
W-11280756 sap concur access token step

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by ramdadi — DISMISSED** [review]

File: sap-concur/4.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -76,32 +76,15 @@ image::sap-concur-config-design.png[config connector]
 |Authorization| A Concur access token.
 |===
 +
-. Create an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method.
-. Configure Basic Auth with your username and password.
+. Create an access token for your Concur instance. Creating an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. To create an access token, use one of the following authentication methods:
```
Comment (dhirsch1) [inline sap-concur/4.0/modules/ROOT/pages/index.adoc]:
```suggestion
. Create an access token for your Concur instance. To do this, specify: `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. 
+
To create an access token, use one of the following authentication methods:
```

Comment (dhirsch1) [reply sap-concur/4.0/modules/ROOT/pages/index.adoc]: "with the GET method is" does not make sense. 
Do you mean "using the GET method ..." or "when the GET method is..."?

Comment (dhirsch1) [reply sap-concur/4.0/modules/ROOT/pages/index.adoc]: Also, I am confused by how the concursolutions.com URL is part of the GET method. Is it in the body? 

What does it mean that the GET method is deprecated? This is what shows in the Preview.

The GET method should be in back ticks.

File: sap-concur/4.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -76,32 +76,15 @@ image::sap-concur-config-design.png[config connector]
...
+* https://developer.concur.com/api-reference/authentication/apidoc.html#auth_grant[Authorization grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#password_grant[Password grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#client_credentials[Client Credentials grant]
+* https://developer.concur.com/api-reference/authentication/apidoc.html#otp_grant[One Time Password grant]
+. After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using Password grant:
```
Comment (dhirsch1) [inline sap-concur/4.0/modules/ROOT/pages/index.adoc]:
```suggestion
. After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using the Password grant:
```

Comment (dhirsch1) [reply sap-concur/4.0/modules/ROOT/pages/index.adoc]: I know using "the" sometimes goes against Kim's advice, but I think this sounds funny when you don't use it.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**